src/container/srv_target.c

```diff
@@ -1443,7 +1445,9 @@ ds_cont_aggregate_ult(void *arg)
 
 		if (dss_xstream_exiting(dmi->dmi_xstream))
 			break;
-		ABT_thread_yield();
+
+		ABT_thread_self(&self);
+		dss_sched_timeout(self, DAOS_AGG_THRESHOLD);
```

Comment:
It's better to have above cont_child_aggregate() to return the 'seconds' for sleep.

Reply:
ok